orca-retrofit/src/main/groovy/com/netflix/spinnaker/orca/retrofit/RetrofitConfiguration.groovy

```diff
@@ -99,18 +98,6 @@ class RetrofitConfiguration {
     return LogLevel.valueOf(retrofitLogLevel)
   }
 
-  /**
```

Comment:
This is a big enough change to warrant it's own separate PR.  I'm pretty sure we've made enough changes so that nothing is throwing RetrofitError anymore, but I'd like to stare at the codebase a fair bit before we remove it.  Removed the unused imports that this PR helps to do that.  And to clarify, retrofit still throws RetrofitError, but since we use SpinnakerRetrofitErrorHandler everywhere, it transforms RetrofitError to Spinnaker*Exception.

Reply:
That makes sense. I am only keeping the changes to remove the unused import statement in this PR. Rest of the changes will be moved to the upcoming PR.